web/src/main/java/com/epam/esm/web/advice/CustomErrorCode.java

```diff
@@ -0,0 +1,18 @@
+package com.epam.esm.web.advice;
+
+public enum CustomErrorCode {
+    RECOURSE_EXIST("404001"),
```

Comment:
please, use the correct error codes, e. g. 
if certificate not found - 404 + 01 = 40401
if tag not found - 404 + 02 = 40402
if certificate validation error - 400 + 01 = 40001 ...

Reply:
done